node/src/main/scala/com/wavesplatform/transaction/transfer/TransferTransaction.scala

```diff
@@ -16,17 +16,17 @@ import play.api.libs.json.JsObject
 import scala.util.Try
 
 case class TransferTransaction(
-    version: TxVersion,
-    sender: PublicKey,
-    recipient: AddressOrAlias,
-    assetId: Asset,
-    amount: TxAmount,
-    feeAssetId: Asset,
-    fee: TxAmount,
-    attachment: ByteStr,
-    timestamp: TxTimestamp,
-    proofs: Proofs,
-    chainId: Byte
+                                version: TxVersion,
+                                sender: PublicKey,
+                                recipient: AddressOrAlias,
+                                assetId: Asset,
+                                amount: TxPositiveAmount,
```

Comment:
No, because of https://github.com/wavesplatform/Waves/blob/version-1.3.x/node/src/main/scala/com/wavesplatform/transaction/validation/impl/TransferTxValidator.scala#L13
https://github.com/wavesplatform/Waves/blob/version-1.3.x/node/src/main/scala/com/wavesplatform/transaction/validation/TxConstraints.scala#L50